### ISSUE TITLE
Bisect tags

### ISSUE THREAD

**Issue by Dagur** [jesseduffield/lazygit#2346]
**Is your feature request related to a problem? Please describe.**
Lazygit has bisect support but you have to select a specific commit. I quickly get lost because there's a lot commits to go through.

**Describe the solution you'd like**
I would like to be able to select a tag and bisect from there

**Comment by mark2185** [comment]
Do I understand correctly, you want to bisect tags?

**Comment by Dagur** [comment]
Yes. I would like to go to the branches panel, select _tags_ and press **b** on an entry there

**Comment by mark2185** [comment]
Ah, so you would just like to _start_ on a tag? Not necessarily _bisect_ only tags?

The easiest way I can currently think of is copying the hash with `ctrl+o` or something, finding it among the commits and going from there.

**Comment by Dagur** [comment]
Maybe a _go to commit_ option on a tag would be useful then

**Comment by mark2185** [comment]
@jesseduffield do we have any mechanism of switching active panel and moving the cursor?

**Comment by jesseduffield** [comment]
@mark2185  it would be something like `self.c.PushContext(context)` and then `context.SetSelectedLineIdx(idx)`

**Comment by mark2185** [comment]
@Dagur are you maybe interesting in taking this up?

Adding a `jump to commit` functionality when browsing tags?

**Comment by Dagur** [comment]
I'll give it a shot but I've never written anything in go 😅

**Comment by Dagur** [comment]
I was able to create a command and focus on the commits using `self.c.PushContext(context)` but that's where I'm stuck. I can't figure out how or where to get the commit index.

**Comment by mark2185** [comment]
Feel free to open up a PR and we can discuss it there :)

**Comment by jesseduffield** [comment]
FWIW you can go `context.GetSelectedIdx()`
